swatch-producer-aws/deploy/clowdapp.yaml

```diff
@@ -85,6 +83,8 @@ objects:
       prometheus: quarkus
   spec:
     envName: ${ENV_NAME}
+    dependencies:
+      - swatch-subscription-sync
```

Comment:
With this configuration, the clowder config file will include the configuration for this service.